### PR TITLE
Put - Post - Patch - Delete

### PR DESCRIPTION
**Pull request by SebastianGomezVidal**
Hi Guys,

Here I did several points toguether (PUT - POST - PATCH - DELETE)

Thanks

### CODE REVIEW

File: test/GithubApi.Put.test.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
+const { StatusCodes } = require('http-status-codes');
```
Comment (kliver98) [inline test/GithubApi.Put.test.js:1]: As comment/suggestion: don't mix a lot of types of implementations.
For example, this file is different from the others such as GithubApi.Issue.test.js or GithubApi.Gist.test.js

File: test/GithubApi.Put.test.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+const { expect } = require('chai');
+const axios = require('axios');
+
+require('dotenv').config({ path: './.env' });
```
Comment (kliver98) [inline test/GithubApi.Put.test.js:5]: This path indicates you have the file `.env` inside test folder. As good programming practice this should be in root or a folder in the root such as config or any other related name

**Review by kliver98 — COMMENTED** [review]
Please do one step by PR in the next ones

File: test/GithubApi.Put.test.js (hunk 1 above)
Comment (SebastianGomezVidal) [reply test/GithubApi.Put.test.js:1]: Ok

File: test/GithubApi.Put.test.js (hunk 2 above)
Comment (SebastianGomezVidal) [reply test/GithubApi.Put.test.js:5]: image
Actually it is no on test, if you read the path it say './' go one folder up and look for the file that .env which is the file were i stored my token but I put .env in git ignore

**Review by kliver98 — APPROVED** [review]

**Pull request merged**